docs/config.yaml

```diff
@@ -91,6 +91,8 @@ streams:
       # OPTIONAL: additional architectures to skip building for
       skip_additional_arches:
         - s390x
+      # OPTIONAL/TEMPORARY: don't upload to govcloud even if cloud is configured
```

Comment:
would be nice to mention here when we can drop it. I know we'd be including information about RHCOS streams here, but it's always nice to leave ourselves breadcrumbs.